core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -23,7 +23,12 @@ import org.broadinstitute.dsde.rawls.serviceperimeter.ServicePerimeterService
 import org.broadinstitute.dsde.rawls.user.UserService._
 import org.broadinstitute.dsde.rawls.util.{FutureSupport, RoleSupport, UserWiths}
 import org.broadinstitute.dsde.rawls.{RawlsException, RawlsExceptionWithErrorReport, StringValidationUtils}
-import org.broadinstitute.dsde.workbench.model.WorkbenchEmail
+import org.broadinstitute.dsde.workbench.model.{
+  Notifications,
+  WorkbenchEmail,
```

Comment:
I'm confused about these imports, as the file didn't change.

Reply:
This PR originally also had the new batchUpdateMembers endpoint in it too, but I split that out because it made the PR too large. Looks like I missed a few imports when pulling out the changes. I will give the PR another look through to make sure I didn't miss anything else.

Cleaned up these imports and the other one you called out in 20a45ead8e7bb9f75bf98f88e30ab2e73ba4a321